src/main/java/com/tieumy/exercise/OOP/Exercise01/Rectangle.java

```diff
@@ -0,0 +1,48 @@
+package com.tieumy.exercise.OOP.Exercise01;
+
+public class Rectangle implements Shape {
+    private final Point topRight;
+
+
+    private final double length;
```

Comment:
Cái này chắc là height

Reply:
Trong toán em thấy người ta kí hiệu là width/breadth x length á anh

Reply:
Công nhận, nhưng bên tin học là width, height

https://learn.microsoft.com/en-us/dotnet/api/system.windows.shapes.rectangle?view=windowsdesktop-7.0

https://docs.oracle.com/javase/7/docs/api/java/awt/Rectangle.html

https://reference.aspose.com/slides/cpp/class/system.drawing.rectangle/

Reply:
DONE